README.md

```diff
@@ -80,7 +80,12 @@ suggestions:
 </ol>
 ```
 
-In fact, such HTML responses are not limited to any particular markup: The
+Note that in this case, selecting an item from within the list will populate
+the search field with the text content of that item. This is necessary in order
+for the component to be usable for assistive technologies and is in accordance
+with general expectations of how an autocomplete component should behave.
```

Comment:
Does this paragraph add value for users here? I'm wary of burdening (potential) users with too much information, especially in this context where they just wanna get something working ASAP.

Perhaps it's time to add a section on design decision towards the bottom of this document? It's certainly a valuable - and perhaps controversial - insight, so worth making explicit.

Reply:
This is a good point. I think that the documentation in this MR would fit better in the Accessibility subsection that was introduced in https://github.com/innoq/simplete/pull/8

That will reduce the documentation for getting started but still allow the user to get an idea about how to make the component as accessible as possible.